src/components/__tests__/RouteListDisplay.cy.js

```diff
@@ -1,21 +1,28 @@
-import { colors } from 'quasar';
+import { colors, date } from 'quasar';
+import { computed } from 'vue';
 import RouteListDisplay from 'components/routes/RouteListDisplay.vue';
 import { i18n } from '../../boot/i18n';
 import { testRouteListDayDate } from '../../../test/cypress/support/commonTests';
 import { useRoutes } from '../../../src/composables/useRoutes';
-import { TransportDirection } from '../../../src/components/types/Route';
+import { useLogRoutes } from '../../../src/composables/useLogRoutes';
+import { rideToWorkByBikeConfig } from '../../../src/boot/global_vars';
 
 const { getPaletteColor } = colors;
 const grey10 = getPaletteColor('grey-10');
 const { getTransportLabel } = useRoutes();
 
+// variables
+const { challengeLoggingWindowDays, challengeStartDate } =
+  rideToWorkByBikeConfig;
+
 describe('<RouteListDisplay>', () => {
   it('has translation for all strings', () => {
     cy.testLanguageStringsInContext([], 'routes', i18n);
   });
 
   context('desktop', () => {
     beforeEach(() => {
+      cy.clock(new Date('2024-08-15').getTime());
```

Comment:
Date literal string repeat across code, you can use const instead of string literal.